core/src/main/resources/application.yml

```diff
@@ -91,9 +91,9 @@ feast:
 
     authorization:
       enabled: false
-      provider: none
+      provider: http
       options:
-        basePath: http://localhost:3000
+        externalAuthUrl: http://localhost:8082
```

Comment:
maybe `authorizationUrl` would be more clear?

Reply:
I added the external because the configuration options are shared across all providers, and it might not be clear that we are talking about a downstream service as opposed to an endpoint on Feast. I dont feel too strongly about this though.